tools/coredumper/sof-coredump-to-gdb.sh

```diff
@@ -17,6 +17,7 @@ if [[ ${reader_result} -ne 0 ]] ; then
 else
 	(echo "${reader_output}")
 	echo "quit" >> coredump-reader-output.txt
-	xt-gdb -q "${elf}" --command=coredump-reader-output.txt
+	xt-gdb -q "${elf}" --command=coredump-reader-output.txt \
+		|& grep -E '^[0-9]|#|\$'
```

Comment:
Can you add some flag to make it verbose (like it was before this PR)?
It looks like you not only filter out *gdb spam* but also possible errors.

Reply:
Thats because there is also one more _spam_ warning so grep for both stdout and stderr will hide it but you are right, we will lose not expected errors.
Im not sure about extra flag for simple example script (which was made to show how this works), maybe better solution will be just to grep only stdout and all errors&warning will be visible. We will see only one repeating warning (one line) but still hide over 13 lines of not important informations (like "Quit anyway? (y or n)")

Reply:
Still would like to have option to get full output -  I see 2 ways: verbose flag or logfile that gets everything

Reply:
Logfile looks good, added